responsive-test-utils/build.gradle.kts

```diff
@@ -20,6 +20,12 @@ plugins {
 
 dependencies {
     implementation(project(":kafka-client"))
+    implementation(libs.kafka.streams.test.utils)
+    implementation(libs.kafka.clients) {
+        artifact {
+            classifier = "test"
+        }
```

Comment:
what does this do? are we depending on the test module only of the kafka.clients?

Reply:
yeah, it's for the MockAdminClient -- seems to be published in a separate jar in https://repo.maven.apache.org/maven2/org/apache/kafka/kafka-clients/3.4.0/